package.json

```diff
@@ -61,6 +61,7 @@
     "babel-eslint": "10.0.2",
     "babel-loader": "7.1.0",
     "babel-plugin-transform-object-rest-spread": "6.26.0",
+    "babel-plugin-transform-require-context": "^0.1.1",
```

Comment:
should I pin this?

Reply:
Looks like in this repo we pin the babel stuff, so yes — we may as well keep that consistent.  (Since they are usually pretty good about semver and also release super frequently, we usually use `^` in other repos.)
```suggestion
    "babel-plugin-transform-require-context": "0.1.1",
```